utils/utils.go

```diff
@@ -171,6 +175,23 @@ func GetIdentifiers(args *skel.CmdArgs) (workloadID string, orchestratorID strin
 	return workloadID, orchestratorID, nil
 }
 
+func GetHandleID(netName string, containerID string, workload string) (string, error) {
+	hasher := md5.New()
```

Comment:
Don't use MD5, it's broken.  If you need a crypto hash, reach for a SHA-2 family function.

Reply:
Do we actually need to hash the network name?  It'd be really nice to avoid that if we can since it makes it a real pain to correlate later.

Reply:
@fasaxc this wasn't for security, it was just to ensure we ended up with only valid characters in the network name portion, as the CNI spec only dictates that it is a string. My thought was instead of trying to escape all invalid characters we just hash it as this value isn't presented to users (correct me if I'm wrong there).

If we really feel it would be helpful to store this value in a human readable format we can do that, we just need to escape the string.

Reply:
@fasaxc @caseydavenport thinking about this a bit more, are we worried about invalid values in `conf.Name`? Is the `HandleID` used anywhere but the IPAM code?